actors/builtin/multisig/multisig_actor.go

```diff
@@ -384,7 +409,7 @@ func (a Actor) ChangeNumApprovalsThreshold(rt vmr.Runtime, params *ChangeNumAppr
 	return nil
 }
 
-func (a Actor) approveTransaction(rt vmr.Runtime, txnID TxnID, txn *Transaction) (bool, []byte, exitcode.ExitCode) {
+func (a Actor) approveTransaction(rt vmr.Runtime, resolvedApprover addr.Address, txnID TxnID, txn *Transaction) (bool, []byte, exitcode.ExitCode) {
```

Comment:
Should AssertMsg that `resolvedApprover` is an ID address

Reply:
This is done.

Reply:
It was unnecessary to resolve the caller in the first place, so also unnecessary to pass it as a parameter here or assert anything.